code/stable/dblpendulum/Website/DblPendulum_SRS.html

```diff
@@ -388,6 +391,14 @@ <h2>Scope of Requirements</h2>
             </p>
           </div>
         </div>
+        <div id="Sec:ReaderChars">
+          <div class="subsection">
+            <h2>Characteristics of Intended Reader</h2>
+            <p class="paragraph">
+              Reviewers of this documentation should have an understanding of undergraduate physics, undergraduate calculus, and ordinary differential equations. The users of Pendulum can have a lower level of expertise, as explained in <a href=#Sec:UserChars>Sec:User Characteristics</a>.
```

Comment:
As mentioned in #2632, I think we should say what level of undergraduate physics and calculus.  I think we should say level 2 physics and level 1 calculus.  I think in the future we should say what topics from physics and calculus we cover, but it is better if we don't get distracted by that right now.  At some point we should go through each of the examples and more carefully record the knowledge needed to understand the examples.